docs/user-guides/desktop/index.md

```diff
@@ -96,17 +96,17 @@ Before you can use Coder Desktop, you will need to sign in.
 
    <Image height="350px" src="../../images/user-guides/desktop/mac-allow-vpn.png" alt="Copy session token" align="center" />
 
-1. Select the Coder icon in the menu bar (macOS) or system tray (Windows), and click the CoderVPN toggle to start the VPN.
+1. Select the Coder icon in the menu bar (macOS) or system tray (Windows), and click the **Coder Connect** toggle to enable the connection.
 
    This may take a few moments, as Coder Desktop will download the necessary components from the Coder server if they have been updated.
 
-1. macOS: You may be prompted to enter your password to allow CoderVPN to start.
+1. macOS: You may be prompted to enter your password to allow Coder Connect to start.
 
-1. CoderVPN is now running!
+1. Coder Desktop is now running!
```

Comment:
```suggestion
1. Coder Connect is now running!
```
?

Reply:
I think you're right - Desktop was already running and now they've got Connect running 🙏

Reply:
we can replace it with what the user should (or can) do next

Reply:
The next step could be to connect to one of the workspaces over ssh.